Source/Scene/QuadtreePrimitive.js

```diff
@@ -496,7 +496,13 @@ define([
             }
         }
 
-        raiseTileLoadProgressEvent(primitive);
+        frameState.afterRender.push(createTileProgressFunction(primitive));
+    }
+
+    function createTileProgressFunction(primitive) {
+        return function() {
+            raiseTileLoadProgressEvent(primitive);
+        }
```

Comment:
You have a `jsHint` error because you're missing a `;` here.
You can run jsHint locally with `npm run jsHint`